src/s3/processor.rs

```diff
@@ -72,10 +73,21 @@ async fn main() -> Result<(), Box<dyn std::error::Error>> {
         let secret = config.aws_secret.as_str();
         let creds = AwsCredentials::new(key, secret, None, None);
 
+        let region_name = config.aws_region.as_str();
+        let region = match Region::from_str(region_name) {
+            Ok(region) => region,
+            Err(_) => {
+                Region::Custom {
+                    name: region_name.to_owned(),
```

Comment:
region_nameをStringで扱ったほうがスッキリしそう
```suggestion
        let region_name = config.aws_region;
        let region = match Region::from_str(&region_name) {
            Ok(region) => region,
            Err(_) => {
                Region::Custom {
                    name: region_name,
```